Lite-Specification/openvg_lite_spec.md

```diff
@@ -28,6 +28,40 @@ Editors
 
 <div style="page-break-after: always;"> </div>
 
+Copyright 2005-2020 The Khronos Group Inc. 
+
+This Specification is protected by copyright laws and contains material proprietary 
+to Khronos. Except as described by these terms, it or any components 
+may not be reproduced, republished, distributed, transmitted, displayed, broadcast 
+or otherwise exploited in any manner without the express prior written permission 
+of Khronos. 
+
+Khronos grants a conditional copyright license to use and reproduce the 
+unmodified Specification for any purpose, without fee or royalty, EXCEPT no licenses 
+to any patent, trademark or other intellectual property rights are granted under these
+terms. Parties desiring to implement the Specification and make use of Khronos trademarks 
+in relation to that implementation, and receive reciprocal patent license protection under 
+the Khronos IP Policy must become Adopters and confirm the implementation as conformant under 
+the process defined by Khronos for this Specification; see https://www.khronos.org/adopters.
+
+Khronos makes no, and expressly disclaims any, representations or warranties, 
+express or implied, regarding this Specification, including, without limitation: 
+merchantability, fitness for a particular purpose, non-infringement of any 
+intellectual property, correctness, accuracy, completeness, timeliness, and 
+reliability. Under no circumstances will Khronos, or any of its Promoters, 
+Contributors or Members, or their respective partners, officers, directors, 
+employees, agents or representatives be liable for any damages, whether direct, 
+indirect, special or consequential damages for lost revenues, lost profits, or 
+otherwise, arising from or in connection with these materials.
+
+Khronos® is a registered trademarks, and OpenVG™ is a trademark of The Khronos 
+Group Inc. OpenGL® is a registered trademark and the OpenGL ES™ logos is a trademark of 
```

Comment:
"OpenGL ES logos" probably shouldn't be pluralized.

Reply:
Agreed - cut and paste error